sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/ParquetCompressionCodecMapper.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.datasources.parquet;
+
+import java.util.Locale;
+
+import org.apache.parquet.hadoop.metadata.CompressionCodecName;
+
+/**
+ * A mapper class easy to obtain compression codecs based on their names.
+ */
+public enum ParquetCompressionCodecMapper {
+  NONE(null),
+  UNCOMPRESSED(CompressionCodecName.UNCOMPRESSED),
+  SNAPPY(CompressionCodecName.SNAPPY),
+  GZIP(CompressionCodecName.GZIP),
+  LZO(CompressionCodecName.LZO),
+  BROTLI(CompressionCodecName.BROTLI),
+  LZ4(CompressionCodecName.LZ4),
+  ZSTD(CompressionCodecName.ZSTD);
+
+  // There are some parquet supported compression codec that doesn't supported by Spark.
+  // LZ4_RAW(CompressionCodecName.LZ4_RAW)
```

Comment:
https://github.com/apache/spark/pull/41507

IIRC, LZ4_RAW is already supported？

Reply:
Thank you for the reminder. I will check it.